Directory.Build.targets

```diff
@@ -0,0 +1,60 @@
+<Project>
+  <ItemGroup Condition="'$(IncludeGlobalAssemblyInfo)' == 'true'">
+    <PackageReference Include="GitInfo" Version="2.0.20" PrivateAssets="All" />
+		<Compile Include="..\Xamarin.Forms.Core\Properties\GlobalAssemblyInfo.cs" Link="Properties\GlobalAssemblyInfo.cs" />
+	</ItemGroup>
+
+  	<!-- Local builds should always be commit count +1, since we'll be always working on the next version to be published. -->
+	<Target Name="_SetLocalCommits" BeforeTargets="_GitPopulateVersionInfo" Condition="'$(CI)' != 'true'">
+		<PropertyGroup>
+			<GitCommits>$([MSBuild]::Add('$(GitCommits)', '1'))</GitCommits>
+		</PropertyGroup>
+	</Target>
+
+  <Target Name="SetGitBranch" AfterTargets="GitInfo" Condition="'$(CI)' == 'true'">
+    <PropertyGroup>
+      <!-- Adjust branch name to CI provided value if available -->
+      <GitBranch Condition="'$(SYSTEM_PULLREQUEST_TARGETBRANCH)' != ''">$(SYSTEM_PULLREQUEST_TARGETBRANCH)</GitBranch>
+      <GitBranch Condition="'$(SYSTEM_PULLREQUEST_TARGETBRANCH)' == '' and '$(BUILD_SOURCEBRANCHNAME)' != ''">$(BUILD_SOURCEBRANCHNAME)</GitBranch>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="GetVersion" BeforeTargets="Build;Pack;GetAssemblyVersion;GetPackageVersion" DependsOnTargets="GitVersion" Returns="$(Version)">
+		<PropertyGroup>
+			<!-- Our version is always the base version, with different pre-release labels for non-baseline builds -->
```

Comment:
This comment clearly contradicts the line just below, where you're using *not* the `BaseVersion` but the semver.